.releaserc

```diff
@@ -0,0 +1,17 @@
+{
+  "branches": ["release/*"],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/github",
+      {
+        "assets": [
+          { "path": "./build/Bip32Ed25519-JVM.jar", "label": "JVM Library" },
+          { "path": "./build/Bip32Ed25519-Android-release.aar", "label": "Android Library" }
+        ]
+      }
+    ],
+    "@semantic-release/git"
```

Comment:
It would be good to consolidate the two release files into a single `releaserc` and use `@semantic-release/git` to handle any staged changes.

- [Example Usage](https://github.com/algorandfoundation/TEALScript/blob/d7d4e8c4bebd4de1a384be73239e0de2e799eb0e/package.json#L126)